p4rt_app/utils/table_utility_test.cc

```diff
@@ -35,6 +35,8 @@ TEST(GetTableType, ReturnsAclForSaiAclAnnotation) {
 
 TEST(GetTableType, ReturnsFixedForNoAnnotation) {
   pdpi::IrTableDefinition ir_table;
+  google::protobuf::TextFormat::ParseFromString(
+        R"pb(preamble { alias: "router_interface_table"})pb", &ir_table);
```

Comment:
The test definition/intent and the addition here is not aligning. The test was meant to ensure that no annotation returns fixed but we are now mandating fixed are part of a defined set.

Reply:
Before there were only ACL and FIXED tables. This test had assumed that something that does not have "@sai_acl" annotation would mean Fixed only. With support of extension tables, absence of any annotation can't be assumed as Fixed, and hence using one of the fixed tables explicitly here.